### PR TITLE
feat: add `MyHistory` aside

### PR DESCRIPTION
**Pull request by diegopf**
EX-6512

EX-6565

### CODE REVIEW

File: src/i18n/messages/es.messages.json [hunk 1]
```diff
@@ -12,6 +12,13 @@
       "removeLabel": "Borrar {suggestion} del historial",
       "title": "HISTORIAL"
     },
+    "myHistory": {
+      "title": "Mi historial",
+      "subtitle": "Estos son tus datos de actividad de búsqueda.",
+      "message": "<span>Son guardados localmente, solo en tu navegador, y nunca salen de ahí.</span><span>Puedes desactivar esta característica más abajo.</span><span class=\"x-font-color--neutral-10 x-font-weight--bold\">Tu privacidad bajo tu control.</span>",
+      "noHistory": "No hay historial de actividad para mostrar",
+      "openButton": "My historial"
```
Comment (mnavarroespinosa) [inline src/i18n/messages/es.messages.json:20]:
```suggestion
      "openButton": "Mi Historial"
```

File: src/i18n/messages/es.messages.json [hunk 2]
```diff
@@ -12,6 +12,13 @@
       "removeLabel": "Borrar {suggestion} del historial",
       "title": "HISTORIAL"
     },
+    "myHistory": {
+      "title": "Mi historial",
```
Comment (mnavarroespinosa) [inline src/i18n/messages/es.messages.json:16]:
```suggestion
      "title": "Mi Historial",
```

File: src/i18n/messages/en.messages.json [hunk 3]
```diff
@@ -12,6 +12,13 @@
       "removeLabel": "Remove {suggestion} from history queries",
       "title": "HISTORY QUERIES"
     },
+    "myHistory": {
+      "title": "My History",
+      "subtitle": "This is your search activity data",
+      "message": "<span>It is stored locally, only in your browser, and never travels outside of it.</span><span>You can choose to disable this option below.</span><span class=\"x-font-color--neutral-10 x-font-weight--bold\">Your privacy under your control.</span>",
+      "noHistory": "No history activity to show",
+      "openButton": "My history"
```
Comment (mnavarroespinosa) [inline src/i18n/messages/en.messages.json:20]:
```suggestion
      "openButton": "My History"
```

File: src/components/predictive-layer/predictive-layer.vue [hunk 4]
```diff
@@ -35,6 +35,14 @@
             : 'x-row__item--span-5 x-padding--right-00'
         ]"
       >
+        <BaseIdModalOpen
+          v-if="$x.device === 'mobile'"
+          modalId="my-history-aside"
+          class="x-button--ghost x-self-end x-padding--right-03"
```
Comment (mnavarroespinosa) [inline src/components/predictive-layer/predictive-layer.vue:41]: The button icon is a bit misaligned due to an invisible border

```suggestion
          class="x-button--ghost x-self-end x-padding--right-03 x-border-width--00"
```

Comment (LuisMartinez15) [reply src/components/predictive-layer/predictive-layer.vue:41]: good catch!

Comment (LuisMartinez15) [reply src/components/predictive-layer/predictive-layer.vue:41]: good catch!

File: src/components/my-history/custom-my-history.vue [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+      HistoryIcon,
+      MyHistory,
+      MyHistoryIcon,
+      MyHistoryIconBw,
+      NoHistoryIcon
+    }
+  })
+  export default class CustomMyHistory extends Vue {
+    @Prop({ default: true })
+    public isMyHistoryEnabled!: boolean;
```
Comment (LuisMartinez15) [inline src/components/my-history/custom-my-history.vue:112]: Can we animate the list of history queries?

```typescript
    protected animation = StaggeredFadeAndSlide;
```

File: src/components/my-history/custom-my-history.vue [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+        <h2 class="x-title2 x-font-weight--bold x-font-color--neutral-10">
+          {{ $t('myHistory.subtitle') }}
+        </h2>
+        <div
+          v-html="$t('myHistory.message')"
+          class="x-list x-list--gap-05 x-title3 x-font-color--neutral-35"
+        ></div>
+      </div>
+      <div class="x-list x-list__item--expand x-border-color--neutral-95 x-border-width--top-01">
+        <MyHistory v-if="isMyHistoryEnabled && $x.fullHistoryQueries.length">
```
Comment (LuisMartinez15) [inline src/components/my-history/custom-my-history.vue:37]:
```suggestion
        <MyHistory v-if="isMyHistoryEnabled && $x.fullHistoryQueries.length" :animation="animation">
```

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
good job!

File: src/components/my-history/custom-my-history.vue (hunk 5 above)
Comment (mnavarroespinosa) [reply src/components/my-history/custom-my-history.vue:114]: Sure, good point! I've added the animation

**Review by LuisMartinez15 — APPROVED** [review]

**Review by mnavarroespinosa — APPROVED** [review]

**Pull request merged**